src/edu/wright/cs/raiderplanner/model/Task.java

```diff
@@ -289,6 +310,28 @@ public void toggleComplete() {
 		}
 	}
 
+	/**
+	 * Toggle one week notification sent.
+	 */
+	public void toggleWeekNotification() {
+		if (this.weekNotificationSent()) {
+			this.weekNotification = false;
+		} else {
+			this.weekNotification = true;
+		}
```

Comment:
This if block could be replaced with `this.weekNotification = !this.weekNotificationSent()`. Both work just fine though.